engine/src/Utils/ColorUtils.ts

```diff
@@ -14,7 +14,6 @@ import { itemFromArray } from "./Utils";
 import { Constants } from "./Constants";
 import { mix, randomInRange, setRangeValue } from "./NumberUtils";
 import type { IColorAnimation } from "../Options/Interfaces/IColorAnimation";
-import type { IParticleValueAnimation } from "../Core/Interfaces";
```

Comment:
This import is not duplicate, if you remove it the build breaks

Reply:
oh okay, it was mentioned by sonarlint